kbfsgit/runner.go

```diff
@@ -1111,13 +1127,14 @@ func (r *runner) pushAll(ctx context.Context, fs *libfs.FS) error {
 	}
 
 	// Finally, packed refs if it exists.
-	_, err = localFS.Stat("packed-refs")
+	_, err = localFS.Stat(packedRefsPath)
 	if os.IsNotExist(err) {
 		return nil
 	} else if err != nil {
 		return err
 	}
-	return r.copyFileWithCount(ctx, localFS, fs, "packed-refs",
+
+	return r.copyFileWithCount(ctx, localFS, fs, packedRefsPath,
```

Comment:
I see a `recursiveCopyWithCounts` call on `fsObjects` above, so this only includes the refs, so it should be fairly small right?

Reply:
Yeah, first we do objects, then the loose refs, and then this is the packed-refs file if one exists.  It would probably less than 1 MB in practice I'd guess.

Reply:
I see; thanks!